modules/ROOT/pages/dw-dates-functions-time.adoc

```diff
@@ -40,7 +40,7 @@ import * from dw::core::Dates
 output application/json
 ---
 {
-  newTime: time({ hour: 12, minutes: 30, seconds: 40 , timeZone: |-03:00|})
+  newTime: time({ hour: 12, minutes: 30, seconds: 40 , timeZone: &#124;-03:00&#124;})
```

Comment:
TODO: CHECK if the HTML code gets transformed to a pipe in this example, prob not.